week08_pomdp/practice_tensorflow.ipynb

```diff
@@ -132,66 +138,77 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from keras.layers import Conv2D, Dense, Flatten\n",
-    "\n",
-    "\n",
-    "class FeedforwardAgent:\n",
```

Comment:
Why is `FeedforwardAgent` gone now?

Reply:
There's no such architecture in pytorch version, so I thought it's excess here